ddtrace/llmobs/_llmobs.py

```diff
@@ -685,14 +685,24 @@ def submit_evaluation(
         if not label:
             log.warning("label must be the specified name of the evaluation metric.")
             return
+
         if not metric_type or metric_type.lower() not in ("categorical", "numerical", "score"):
-            log.warning("metric_type must be one of 'categorical', 'numerical', or 'score'.")
+            log.warning("metric_type must be one of 'categorical' or 'score'.")
             return
+
+        metric_type = metric_type.lower()
+        if metric_type == "numerical":
+            log.warning(
```

Comment:
for usability it's good to include what version it will go away in if possible. A lot of people take "deprecated" as it will stick around for quite a while and not take action.

Reply:
Updated the log to mention what version it will be deprecated in, let me know if "ddtrace 2.10.x" is not the correct terminology here

Reply:
oh we should say when it will be remove and that it is already deprecated.

eg: "The evaluation metric type 'numerical' is deprecated and will be removed in ddtrace 2.11. Use 'score' instead."

Reply:
nit: Would it make sense to use `from ddtrace.vendor.debtcollector import deprecate` here instead of `log.warn`?

Reply:
Sorry the deprecation release note/ pr description was misleading - we're not actually deprecating anything here since the `numerical` metric type was never supported and would be dropped by our backend. This PR is more of a bug fix + adds a warning message for users who ended up using an unsupported metric type. I updated the pr description to be more clear of this!